docs/charts.md

```diff
@@ -25,6 +25,7 @@ wordpress/
   Chart.yaml        # A YAML file containing information about the chart
   LICENSE           # OPTIONAL: A plain text file containing the license for the chart
   README.md         # OPTIONAL: A human-readable README file
+  NOTES.txt         # OPTIONAL: A plain text file containing short usage notes
```

Comment:
Gotcha. No strong objections, just asking since LICENSE is also a plain text file but without an extension, so I think it would make it look a bit more consistent without it.